parteh/PRTGenericMod.F90

```diff
@@ -279,7 +279,7 @@ module PRTGenericMod
                                                         ! examples are carbon12_element
                                                         ! nitrogen_element, etc.
 
-  integer, public :: element_pos(num_organ_types)       ! This is the reverse lookup
+  integer, public :: element_pos(num_element_types)     ! This is the reverse lookup
```

Comment:
This is just changed to align the type correct?  I.e. it should have been elements, but since the value (6) is the same, it was never an issue?

Reply:
yeah, it worked ok because we have at least as many organs as element types